### DISCUSSION TITLE
clearAmbientCache says deprecated in v10

### DISCUSSION THREAD

**Discussion by nafeelcassim** [rnmapbox/maps#2970] [Q&A]
When i try to use clearAmbientCache Method it says its deprecated , i'm using -> "@rnmapbox/maps": "^10.0.9-rc.0", whats the alternative if its deprecated, or is it handled internally. Also  when i load the map normally does it use the ambient cache ? . Or is ambient cache used only with tiles

**Comment by mmarie4** [comment]
Did you find any solution ? I still have the issue.

I am using offline manager and deleting packs doesn't free any storage on the device...
